### ISSUE TITLE
Add support for flight logs

### ISSUE THREAD

**Issue by pierotofy** [uav4geo/DroneDB#12]
Relevant discussion: https://community.opendronemap.org/t/new-project-dronedb-request-for-thoughts/2968/9

Similar to https://app.airdata.com/

DJI flight logs are encrypted, so analysis is needed to read those.

**Comment by pierotofy** [comment]
Also need to codify what fields are common to most flight logs so that a semi-standard interface can be applied to different vendors.

**Comment by adamsteer** [comment]
@pierotofy there are loads of research data managers working on this, but not much visible output. Have a dig though:

https://www.rd-alliance.org/groups/small-unmanned-aircraft-systems%E2%80%99-data-ig

**Comment by pierotofy** [comment]
Yep. Data management is an unsolved problem. Lots of ideas, but few general purpose implementations. Thanks for the link!

**Comment by adamsteer** [comment]
exactly! you'll see that in all the discussions about data management - lots of problem space descriptions. I think this project has a really strong role in *actually making things happen*, so I'm pushing that barrow a lot ;)

**Comment by Saijin-Naib** [comment]
This could be very helpful for ArduPilot/3DR Solo/PX4 logs.
https://github.com/ArduPilot/UAVLogViewer
